python/py-nibabel/Portfile

```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-nibabel
-version                 2.3.3
+version                 2.4.0
 revision                0
 platforms               darwin
 license                 MIT
 maintainers             nomaintainer
 description             Access a cacophony of neuro-imaging file formats
 
-long_description        Nibabel provides read and write access to some common medical and \
-    neuroimaging file formats, including: ANALYZE (plain, SPM99, SPM2), GIFTI, NIfTI1, MINC, \
-    MGH and ECAT as well as PAR/REC. We can read and write Freesurfer geometry, and read \
-    Freesurfer morphometry and annotation files. There is some very limited support for \
-    DICOM. NiBabel is the successor of PyNIfTI.
+long_description        Nibabel provides read and write access to some \
+                        common medical and neuroimaging file formats, \
+                        including: ANALYZE (plain, SPM99, SPM2), GIFTI, \
+                        NIfTI1, MINC, MGH and ECAT as well as PAR/REC. We \
+                        can read and write Freesurfer geometry, and read \
+                        Freesurfer morphometry and annotation files. There \
+                        is some very limited support for DICOM. NiBabel is \
+                        the successor of PyNIfTI.
 
 homepage                https://nipy.org/nibabel
 master_sites            pypi:n/nibabel/
 distname                nibabel-${version}
 
-checksums               rmd160  1f202d10b89872cb056f64334f262cfa02df0075 \
-                        sha256  b6366634c65b04464e62f3a9a8df1faa172f780ed7f1af1c6818b3dc2f1202c3 \
-                        size    4492243
+checksums \
+    rmd160  a479afa81c5b79752ef0f39dcde0304e53a26f3a \
+    sha256  dd0c41715d0391c724e2828bba2c16690dbd6aafbca8e920ee8448ed0086e4c1 \
+    size    4589348
 
 python.versions         27 34 35 36 37
 
 if {${name} ne ${subport}} {
+    depends_test-append     port:py${python.version}-nose \
+                            port:py${python.version}-mock \
+                            port:py${python.version}-Sphinx \
+                            port:py${python.version}-pytest \
+                            port:py${python.version}-setuptools
+
     depends_build-append \
                         port:py${python.version}-setuptools
 
     depends_lib-append  port:py${python.version}-numpy \
+                        port:py${python.version}-bz2file \
                         port:py${python.version}-six
 
+    test.run    yes
```

Comment:
it actually doesn't pass all tests with Python 3.7, not sure if that's a "real" problem or not...